test_conformance/vulkan/vulkan_interop_common/vulkan_wrapper.cpp

```diff
@@ -615,6 +615,17 @@ VulkanQueue::VulkanQueue(VkQueue vkQueue): m_vkQueue(vkQueue) {}
 
 VulkanQueue::~VulkanQueue() {}
 
+void VulkanQueue::submit(const VulkanCommandBuffer &commandBuffer,
+                         const VkFence &fence)
+{
+    VkSubmitInfo vkSubmitInfo = {};
+    vkSubmitInfo.sType = VK_STRUCTURE_TYPE_SUBMIT_INFO;
+    vkSubmitInfo.pNext = NULL;
+    vkSubmitInfo.waitSemaphoreCount = (uint32_t)0;
```

Comment:
Command buffer links are missing in vkSubmitInfo.
See below code at line number 642 and 643 for reference.
vkSubmitInfo.commandBufferCount = (uint32_t)commandBufferList.size();
    vkSubmitInfo.pCommandBuffers = commandBufferList();